src/devices/video/mc6845.cpp

```diff
@@ -473,23 +490,32 @@ void mc6845_device::recompute_parameters(bool postload)
 			// Also, the mode-register change needs to be added to the changed-parameter tests above.
 			if (MODE_INTERLACE_AND_VIDEO)
 			{
-				//max_visible_y *= 2;
+				//visible_height *= 2;
 				//vert_pix_total *= 2;
 			}
 
 			if(m_show_border_area)
+			{
 				visarea.set(0, horiz_pix_total-2, 0, vert_pix_total-2);
+			}
 			else
-				visarea.set(0 + m_visarea_adjust_min_x, max_visible_x + m_visarea_adjust_max_x, 0 + m_visarea_adjust_min_y, max_visible_y + m_visarea_adjust_max_y);
+			{
+				visarea.set(0 + m_visarea_adjust_min_x, visible_width - 1 + m_visarea_adjust_max_x, 0 + m_visarea_adjust_min_y,
+				 visible_height - 1 + m_visarea_adjust_max_y);
```

Comment:
If you’re going split the line, please indent the continuation by two tabs, and group lines logically, e.g. put the X and Y coordinates on their own lines, like:
```c++
			{
				visarea.set(
							0 + m_visarea_adjust_min_x, visible_width - 1 + m_visarea_adjust_max_x,
							0 + m_visarea_adjust_min_y, visible_height - 1 + m_visarea_adjust_max_y);
			}
```